### ISSUE TITLE
URI translation

### ISSUE THREAD

**Issue by flo-sch** [rainlab/translate-plugin#112]
Hey,

Small question about this plugin,
Is it possible, or is there any plan in the future to enable URI-translation?

Basically, if I have three languages, for instance "en", "fr", "de", and an about page,
I currently can access :

```
/en/about
/fr/about
/de/about
```

For SEO purposes, I would prefer to access (as the same route) :

```
/en/about
/fr/a-propos
/de/uber-uns
```

**Comment by CptMeatball** [comment]
Duplicate of #25.

**Comment by Flo-Schield-Bobby** [comment]
Indeed, didn't see that one, sorry.
Should I close this one?

**Comment by daftspunk** [comment]
Merged with #25
